### DISCUSSION TITLE
Question about UI when it migrated from v1 to v2

### DISCUSSION THREAD

**Discussion by guiferpa** [twilio-labs/paste#3443] [General] [answered]
Currently I changed Flex UI from v1 to v2 at Flex Dashboard and I configured all plugins to use `@twilio/paste`. However I figured out that the Dashboard UI still using MUI instead of Paste UI. It's a problem for teams that uses Paste as UI library to implements plugins because of different styles between MUI and Paste. I'd like know how I can  add Paste UI to works at all components without exceptions.

**Follow a pic as example**

🎈  In this pic there's a button that uses Paste UI as style provider and the rest of components uses still. using MUI.

<img width="397" alt="Screenshot 2023-08-24 at 18 02 40" src="https://github.com/twilio-labs/paste/assets/9096630/87a55aed-2d62-48fe-8a4f-48b44e62b3f5">

**Comment by serifluous** [comment, marked as answer]
Hey @guiferpa ! Can you share a few more examples of where you're facing style differences?

We may be able to help with styling issues, but in some cases, we might have to redirect you to the Flex support team. The MUI components in Flex right now are designed to visually pair well with Paste components, and eventually in a future version will be fully replaced by Paste components.

For the example you shared, you can swap your button to the [inverse button](https://paste-storybook.twilio.design/?path=/story/components-button--inverse-button) or [inverse link button](https://paste-storybook.twilio.design/?path=/story/components-button--inverse-link-button) by setting `variant="inverse"` or `variant="inverse_link"`.

**Reply by guiferpa** [reply]
Hey @serifluous many thanks for your time. That's enough, you saying that currently you working to compatibility between MUI and Paste sounds good for my question. I Ithought that Flex Dashboard components were full migrated from MUI to Paste UI. Is there some milestone to follow about full Paste UI implementation in Flex Dashboard components?

**Reply by serifluous** [reply]
I'd recommend reaching out to Flex support. They'll have the most accurate information for you on a timeline!

Let us know if you run into other styling issues in the future. There are some that we might be able to help solve before Flex is fully built with Paste components.
